File: C++/11720-숫자의 합.cpp
#include <iostream>
using namespace std;

int main (){
    int n, sum = 0;
    char a;
    
    cin>>n;
    
    for(int i = 0; i < n; i++){
        cin>>a;
        sum+= a - '0';
    }
    
    cout<<sum<<endl;
    return 0;
}

File: Java/11720-숫자의 합.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        int sum = 0;

        int n = sc.nextInt();
        String a = sc.next();

        for(int i = 0; i < n; i++){
            sum += a.charAt(i) - '0';
        }

        System.out.println(sum);
        sc.close();
    }
}
